keras_cv/models/backbones/mobilenet_v3/mobilenet_v3_backbone.py

```diff
@@ -0,0 +1,437 @@
+            layer.
+        input_shape: optional shape tuple, defaults to (None, None, 3).
+        input_tensor: optional Keras tensor (i.e., output of `layers.Input()`)
+            to use as image input for the model.
+        alpha: float, controls the width of the network. This is known as the
+            depth multiplier in the MobileNetV3 paper, but the name is kept for
+            consistency with MobileNetV1 in Keras.
+            - If `alpha` < 1.0, proportionally decreases the number
+                of filters in each layer.
+            - If `alpha` > 1.0, proportionally increases the number
+                of filters in each layer.
+            - If `alpha` = 1, default number of filters from the paper
+                are used at each layer.
+        dropout_rate: a float between 0 and 1 denoting the fraction of input
+            units to drop, defaults to 0.2.
+
+    Examples:
+    ```python
+    input_data = tf.ones(shape=(8, 224, 224, 3))
+
+    # Randomly initialized backbone with a custom config
+    model = MobileNetV3Backbone(
+        stackwise_expansion=[1, 72.0 / 16, 88.0 / 24, 4, 6, 6, 3, 3, 6, 6, 6],
+        stackwise_filters=[16, 24, 24, 40, 40, 40, 48, 48, 96, 96, 96],
+        stackwise_stride=[2, 2, 1, 2, 1, 1, 1, 1, 2, 1, 1],
+        filters=1024,
+        include_rescaling=False,
+    )
+    output = model(input_data)
+    ```
+    """  # noqa: E501
+
+    def __init__(
+        self,
+        *,
+        stackwise_expansion,
+        stackwise_filters,
+        stackwise_stride,
+        filters,
+        include_rescaling,
+        input_shape=(None, None, 3),
+        input_tensor=None,
+        alpha=1.0,
+        dropout_rate=0.2,
+        **kwargs,
+    ):
+        activation = layers.ReLU()
+
+        inputs = utils.parse_model_inputs(input_shape, input_tensor)
+        x = inputs
+
+        if include_rescaling:
+            x = layers.Rescaling(scale=1 / 255)(x)
+
+        x = layers.Conv2D(
+            16,
+            kernel_size=3,
+            strides=(2, 2),
+            padding="same",
+            use_bias=False,
+            name="Conv",
+        )(x)
+        x = layers.BatchNormalization(
+            axis=channel_axis,
+            epsilon=1e-3,
+            momentum=0.999,
+            name="Conv/BatchNorm",
+        )(x)
+        x = activation(x)
+
+        pyramid_level_inputs = {}
+        for stack_index in range(len(stackwise_filters)):
+            x = apply_inverted_res_block(
+                x,
+                expansion=stackwise_expansion[stack_index],
+                filters=adjust_channels(
+                    (stackwise_filters[stack_index]) * alpha
+                ),
+                kernel_size=3,
+                stride=stackwise_stride[stack_index],
+                se_ratio=None,
+                activation=activation,
+                expansion_index=stack_index,
+            )
+            pyramid_level_inputs[stack_index] = x
```

Comment:
Awesome! This looks correct to me in terms of pyramid_level_inputs, although we want to record the layer name instead of the tensor itself, so here we should replace `x` with `x.node.layer.name`